ui/src/app/store/face-verification/reducers.ts

```diff
@@ -34,8 +43,14 @@ const initialStateVerification: FaceVerificationEntityState = {
 
 const reducerVerification: ActionReducer<FaceVerificationEntityState> = createReducer(
   initialStateVerification,
-  on(verifyFace, verifyFaceSuccess, (state, action) => ({ ...state, ...action, isPending: false })),
-  on(verifyFaceReset, verifyFaceFail, () => ({ ...initialStateVerification }))
+  on(verifyFaceAddProcessFile, (state, action) => ({ ...state, ...action })),
+  on(verifyFaceAddCheckFileFile, (state, action) => ({ ...state, ...action })),
```

Comment:
please combine multiple actions in 1 reducer function

Reply:
fixed